zhaquirks/sonoff/s60zbtpf.py

```diff
@@ -0,0 +1,190 @@
+"""Sonoff S60ZBTPF - Zigbee Smart Plug."""
+
+from typing import Final
+
+from zigpy import types
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfPower,
+)
+from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
+import zigpy.types as t
+from zigpy.zcl import ClusterType, foundation
+import zigpy.zcl.foundation as zcl_f
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class SonoffCluster(CustomCluster):
+    """Custom Sonoff cluster."""
+
+    cluster_id = 0xFC11
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        # network_led = ZCLAttributeDef(
+        #     name="network_led",
+        #     id=0x0001,
+        #     type=t.Bool,
+        # )
```

Comment:
Does this attribute not work? If so, it can be removed. We shouldn't add commented code.
If the attribute does work, we can define it here (so just uncomment). We don't have to necessarily add a config entity for all attributes (although it might make sense for this attribute).